### PR TITLE
 French translations

### PR DESCRIPTION
**Pull request by tayre**
First pass of French translations. These are entirely from Google translate. 

I'm hopeful a bilingual speaker can help me improve the quality of these within this PR.

screencapture-localhost-8080-policies-2019-2019-10-10-16_39_50

### CODE REVIEW

File: static/policies/2019/policies.json [hunk 1]
```diff
@@ -614,7 +614,7 @@
     "party": "Conservative",
     "title": {
       "en": "Cut tax rate on lowest income bracket from 15% to 13.75%",
-      "fr": ""
+      "fr": "Réduction du taux d'imposition de la tranche de revenu la plus basse de 15% à 13,75%"
```
Comment (carolinerioux) [inline static/policies/2019/policies.json]:
```suggestion
      "fr": "Réduire le taux d'imposition de la tranche de revenu la plus basse de 15% à 13,75%"
```

File: static/policies/2019/policies.json [hunk 2]
```diff
@@ -631,7 +631,7 @@
     "party": "Liberal",
     "title": {
       "en": "Net-zero emissions by 2050",
-      "fr": ""
+      "fr": "Émissions nettes nettes d'ici 2050"
```
Comment (carolinerioux) [inline static/policies/2019/policies.json]:
```suggestion
      "fr": "Émissions nettes zéro d'ici 2050"
```

File: static/policies/2019/policies.json [hunk 3]
```diff
@@ -742,7 +742,7 @@
     "party": "Conservative",
     "title": {
       "en": "Allow 30-year mortgages for first-time home buyers",
-      "fr": ""
+      "fr": "Autoriser les acheteurs d'une première maison d'une hypothèque de 30 ans"
```
Comment (carolinerioux) [inline static/policies/2019/policies.json]:
```suggestion
      "fr": "Autoriser les acheteurs d'une première maison à obtenir une hypothèque de 30 ans"
```

File: static/policies/2019/policies.json [hunk 4]
```diff
@@ -765,7 +765,7 @@
     "party": "NDP",
     "title": {
       "en": "Allow 30-year mortgages for first-time home buyers",
-      "fr": ""
+      "fr": "Autoriser les acheteurs d'une première maison d'une hypothèque de 30 ans"
```
Comment (carolinerioux) [inline static/policies/2019/policies.json]:
```suggestion
      "fr": "Autoriser les acheteurs d'une première maison à obtenir une hypothèque de 30 ans"
```

File: static/policies/2019/policies.json [hunk 5]
```diff
@@ -822,7 +822,7 @@
     "party": "Liberal",
     "title": {
       "en": "Free energy audits for homeowners and landlords",
-      "fr": ""
+      "fr": "Audits énergétiques gratuits pour les propriétaires et les propriétaires"
```
Comment (carolinerioux) [inline static/policies/2019/policies.json]:
```suggestion
      "fr": "Audits énergétiques gratuits pour les propriétaires"
```

File: static/policies/2019/policies.json [hunk 6]
```diff
@@ -839,7 +839,7 @@
     "party": "Liberal",
     "title": {
       "en": "Interest-free loans up to $40k for green renovations",
-      "fr": ""
+      "fr": "Prêts sans intérêt jusqu'à 40 k $ pour des rénovations vertes"
```
Comment (carolinerioux) [inline static/policies/2019/policies.json]:
```suggestion
      "fr": "Prêts sans intérêt jusqu'à 40 000 $ pour des rénovations vertes"
```

File: static/policies/2019/policies.json [hunk 7]
```diff
@@ -873,7 +873,7 @@
     "party": "Liberal",
     "title": {
       "en": "Invest $100 million to train workers in energy audits, retrofits, and net-zero home construction",
-      "fr": ""
+      "fr": "Investir 100 millions de dollars pour former les travailleurs aux audits énergétiques, à la modernisation et à la construction de maisons nettes"
```
Comment (carolinerioux) [inline static/policies/2019/policies.json]:
```suggestion
      "fr": "Investir 100 millions de dollars pour former les travailleurs aux audits énergétiques, à la modernisation et à la construction de maisons à émission zéro"
```

File: static/policies/2019/policies.json [hunk 8]
```diff
@@ -930,7 +930,7 @@
     "party": "Liberal",
     "title": {
       "en": "1% vacancy and speculation tax for non-resident, non-Canadians",
-      "fr": ""
+      "fr": "taxe de 1% sur la vacance et la spéculation pour les non-résidents et les non-Canadiens"
```
Comment (carolinerioux) [inline static/policies/2019/policies.json]:
```suggestion
      "fr": "taxe de 1% sur l'inoccupation et la spéculation pour les non-résidents et les non-Canadiens"
```

File: static/policies/2019/policies.json [hunk 9]
```diff
@@ -940,7 +940,7 @@
     "party": "NDP",
     "title": {
       "en": "National registry to track individuals behind business entities who own/buy land",
-      "fr": ""
+      "fr": "Registre national pour suivre les individus derrière les entités commerciales qui possèdent / achètent des terres"
```
Comment (carolinerioux) [inline static/policies/2019/policies.json]:
```suggestion
      "fr": "Registre national pour suivre les individus responsables d'entités commerciales qui possèdent / achètent des terres"
```

File: static/policies/2019/policies.json [hunk 10]
```diff
@@ -1017,7 +1017,7 @@
     "party": "Green",
     "title": {
       "en": "Increase federal contribution to Canada Housing Benefit",
-      "fr": ""
+      "fr": "Augmenter la contribution fédérale à l'allocation canadienne de logement"
```
Comment (carolinerioux) [inline static/policies/2019/policies.json]:
```suggestion
      "fr": "Augmenter la contribution fédérale aux prestations relatives au logement du Canada"
```

File: src/support/Content.re [hunk 11]
```diff
@@ -6,44 +6,43 @@ module Strings = {
...
+  let reach = {en: "Reach", fr: {js|Atteindre|js}};
+  let confidence = {en: "Confidence", fr: {js|Confiance|js}};
+  let effort = {en: "Effort", fr: {js|Effort|js}};
 
+  let foreign_policy = {en: "Foreign Policy", fr: {js|La Politique Extérieure|js}};
+  let taxes = {en: "Taxes", fr: {js|Les taxes|js}};
+  let international_trade = {en: "International Trade", fr: {js|Échange international|js}};
+  let environment = {en: "Environment", fr: {js|Environnement|js}};
+  let government = {en: "Government", fr: {js|Gouvernement|js}};
+  let indigenous_relations = {en: "Indigenous Relations", fr: {js|Relations Autochtones|js}};
```
[the remark this answers is not in the document]
Comment (amelieasirois) [reply src/support/Content.re]: To make sure you keep the sentence intent... maybe: 
`les relations avec les peuples autochtones`

**Comment by tayre** [comment]
Much improved French translations. Merci @carolinerioux .

updated_fr_translations

File: src/support/Content.re [hunk 12]
```diff
@@ -6,44 +6,43 @@ module Strings = {
...
-  let cannabis = {en: "Cannabis", fr: {js|?|js}};
-  let social_assistance = {en: "Social Assistance", fr: {js|?|js}};
-  let youth = {en: "Youth", fr: {js|?|js}};
-  let senate = {en: "Senate", fr: {js|?|js}};
-  let electoral_reform = {en: "Electoral Reform", fr: {js|?|js}};
-  let public_safety = {en: "Public Safety", fr: {js|?|js}};
-  let education = {en: "Education", fr: {js|?|js}};
-  let housing = {en: "Housing", fr: {js|?|js}};
-  let affordability = {en: "Affordability", fr: {js|?|js}};
+  let reach = {en: "Reach", fr: {js|Atteindre|js}};
```
[the remark this answers is not in the document]
Comment (jahfer) [reply src/support/Content.re]: These strings are actually not used yet. I had just stuck them in as a placeholder for an idea I had that I never excuted on. I think `Portée` is the relevant one!

**Comment by jahfer** [comment]
Thank you both ❤️

**Pull request merged**